Discord Metadata for specific pages

AsMurphy-student opened the pull request:
## Title: Closes #6  

## Description
As described in the issue I went ahead and added in meta tags for discord embedding. I created one for title, description, and URL. I confirmed that these 4 built correctly locally. I commented out the meta tag for image as I could not find an applicable image. Was not able to test embedding locally with localhost url.

## Checklist
- [x] Added in meta tags at _layouts/default.html
- [x] Created tags for Title, Description, and URL for default
- [x] Confirmed tags build correctly with correct data in _site locally (did **not** push _site of course)
- [x] Updated default metatags to include fallback title and description
- [x] Added in Proper descriptions for page below
- [x] bus.html
- [x] button.html
- [x] docs.html
- [x] elevator.html
- [x] index.html
- [x] succuss.html
- [x] summary.html
- [x] confirmed all changes build successfully
- [x] Changed to instead use the jekyll-seo-tag plugin
- [x] Added discord link in main page
- [x] Added discord button on success page
- [x] Added catalog link with short blurb to direct users to use if they wish to have pre-filled fields

AsMurphy-student committed:
Added in meta tags and checked if they built correctly in _site locally

Commented out meta tag for image as I could not find one that would be applicable; let me know if there is and I will update

MoralCode commented:
Each page type has a way to set its own meta information too. So anything set in the default layout is just going to be a default

AsMurphy-student committed:
Updated metadata title and description to have fallback defaults

AsMurphy-student committed:
Updated all pages to have description to be used in site metadata

These include bus.html, button.html, docs.html, elevator.html, index.html, success.html, and summary.html

MoralCode commented on _layouts/default.html:
```diff
@@ -28,6 +28,13 @@
+	<meta property="og:description" content="{{ page.description | default: 'This is a page for the CampusPulse Reporting System' }}">
+	<!-- Meta tag for image if applicable -->
+	<!-- <meta property="og:image" content="{{ page.image | absolute_url }}" /> -->
+	<meta property="og:url" content="{{ page.url | absolute_url }}" />
```
i think theres a jekyll plugin that can handle this for us and is supported on GH pages (i think its named something like jekyll-seo-tags or something, if you look at the homepage for github.com/classclock or one of my other jekyll based sites youll probably find it

AsMurphy-student committed:
Commented out default.html meta tags to use jekyll-seo-tags

AsMurphy-student committed:
Install jekyll-seo-tag plugin and setup to use in default.html

AsMurphy-student committed:
Commented out old meta tags and added some space at top away from matomo code

AsMurphy-student committed:
feat: added discord link in reporting page

AsMurphy-student committed:
feat: added discord server button on success page

AsMurphy-student committed:
feat: added elevator catalog link in elevator form
